### PR TITLE
feat: set special labels to dependabot PRs

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: set special labels to dependabot PRs

Set release-note-none and ok-to-test labels to
prs which were created by dependabot

**Release note**:
```
NONE
```

### CODE REVIEW

File: .github/workflows/dependabot-ok-to-test.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Add ok-to-test label for dependabot PRs
+on: pull_request
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
```
Comment (akrejcir) [inline .github/workflows/dependabot-ok-to-test.yaml:10]: The metadata is not used anywhere.

Comment (akrejcir) [reply .github/workflows/dependabot-ok-to-test.yaml:10]: I see now, it's explained here: https://github.com/dependabot/fetch-metadata?tab=readme-ov-file#auto-approving

**Review by 0xFelix — APPROVED** [review]
/approve

Btw. if we wanted to create a meaningful release note, IIRC you can also put the release note into the commit message.

**Pull request merged**